tests/e2e/test_rosacli_cluster.go

```diff
@@ -1381,6 +1620,120 @@ var _ = Describe("Create cluster with invalid options will",
 
 			})
 
+		It("to validate the invalid proxy when create cluster - [id:45509]", labels.Medium, labels.Runtime.Day1Negative,
+			func() {
+				zone := constants.CommonAWSRegion + "a"
+				clusterName := "rosacli-45509"
+				By("Create rosa cluster which has proxy without subnets set by command ")
+				output, err, _ := clusterService.Create(
+					"cl-45509",
+					"--http-proxy", "http://example.com",
+					"--https-proxy", "https://example.com",
+				)
+				Expect(err).To(HaveOccurred())
+				Expect(output.String()).Should(ContainSubstring("The number of subnets for a"))
+
+				By("Prepare vpc with subnets")
+				vpc, err := vpc_client.PrepareVPC(clusterName, constants.CommonAWSRegion, "", true, "")
+				Expect(err).ToNot(HaveOccurred())
+				// defer vpc.DeleteVPCChain(true)
```

Comment:
Why don't delete VPC and subnets resources at the end?

Reply:
Oh. forget to comment it back. Updating...

Reply:
Uncommented